workshops/templates/workshops/instructors.html

```diff
@@ -10,7 +10,7 @@
 {% endblock %}
 
 {% block content %}
-    {% crispy form form_helper %}
+    {% crispy form %}
 {% if persons == None %}
```

Comment:
@narayanaditya95:
> ```
{% if persons %}
    {% for p in persons %}
        ## Tables
    {% empty %}
        <p>No matches.</p>
{% endfor %}
```

Yeah, that's what I wanted.

> Should I squash the first four commits into one since it is an atomic update. Is that right?

I'll answer in the comment thread to that.